src/main/java/com/iexec/blockchain/command/task/reveal/TaskRevealService.java

```diff
@@ -35,16 +35,16 @@ public TaskRevealService(
         super(blockchainService, storageService, queueService);
     }
 
-    public String start(String chainTaskId, TaskRevealArgs args) {
+    public String start(String chainTaskId,
+                        com.iexec.common.chain.adapter.args.TaskRevealArgs args) {
```

Comment:
I wish I could remove one.
One is external (DTO is in common), the other is internal (blockchain adapter domain). The internal version keeps track of the chain object id (used as a context).
In a future PR we can try to:
1. Rename internal objects so we don't have com.iexec.common.chain.adapter.args.[..] appearing in the code (Use something like TaskRevealArgsInternal instead of com.iexec.blockchain.command.task.reveal.TaskRevealArgsInternal)
2. Try to remove com.iexec.blockchain.command.task.reveal.Task[..]

Reply:
I see, thanks for the clarification.